build/NuSpecs/WindowsAppSDK-Nuget-Common.targets

```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- Check TargetPlatformMinVersion during build to block unsupported configurations. -->
+  <Target Name="MicrosoftWindowsAppSDKCheckTargetPlatformVersion" BeforeTargets="PrepareForBuild"
+      Condition="'$(TargetPlatformVersion)' != '' and '$(TargetPlatformMinVersion)' != '' and '$(SkipMicrosoftWindowsAppSDKCheckTargetPlatformVersion)'==''">
+    <PropertyGroup>
+      <MicrosoftWindowsAppSDKTargetPlatformMinCheckValue>$([System.Version]::Parse('$(TargetPlatformMinVersion)').Build)</MicrosoftWindowsAppSDKTargetPlatformMinCheckValue>
+    </PropertyGroup>
+    <Error
+        Text="Microsoft.WindowsAppSDK nuget package requires TargetPlatformMinVersion &gt;= 10.0.17763.0 (current project is $(MicrosoftWindowsAppSDKTargetPlatformMinCheckValue))"
```

Comment:
I'm not sure we should be requiring apps to have min versions of 17763.

Apps should be able to choose to dynamically light-up, right? Like if they want to run on 15063 and avoid calling any WinAppSDK APIs unless they detect they're running on 17763 or higher, that should be fine, correct?

Yes, the WinAppSDK APIs themselves won't work below 17763, but just like other new features like a new ColorPicker API, apps should be able to dynamically use them uplevel without cutting off old versions.

Reply:
There was some discussion of this a few months ago but I don't know where it landed. Please follow up with @BenJKuhn.

But it's not a rename issue so good question but no change.